tools/win/generate_breakpad_symbols.py

```diff
@@ -45,94 +72,63 @@ def mkdir_p(path):
     except OSError as e:
         if e.errno == errno.EEXIST and os.path.isdir(path):
             pass
-        else: raise
+        else:
+            raise
 
 
-def GenerateSymbols(options, binaries):
+async def GenerateSymbols(options, binaries):
     """Dumps the symbols of binary and places them in the given directory."""
-
-    q = queue.Queue()
-    print_lock = threading.Lock()
-
-    def _Worker():
-        while True:
-            binary = q.get()
-
-            if options.verbose:
-                with print_lock:
-                    print("Generating symbols for {0}".format(binary), flush=True)
-                    thread_start = datetime.utcnow()
-
-            dump_syms = os.path.join(options.build_dir, 'dump_syms.exe')
-            syms = GetCommandOutput([dump_syms, binary])
-            module_line = re.match(r"MODULE [^ ]+ [^ ]+ ([0-9A-Fa-f]+) (.*)\r\n", syms)
-            if module_line is None:
-                with print_lock:
-                    print("Failed to get symbols for {0}".format(binary), flush=True)
-                q.task_done()
-                continue
-
-            output_path = os.path.join(options.symbols_dir, module_line.group(2),
-                                       module_line.group(1))
-            mkdir_p(output_path)
-            symbol_file = "%s.sym" % module_line.group(2)[:-4]  # strip .pdb
-            f = open(os.path.join(output_path, symbol_file), 'w') # pylint: disable=consider-using-with
-            f.write(syms)
-            f.close()
-
-            if options.verbose:
-                with print_lock:
-                    thread_end = datetime.utcnow()
-                    elapsed = thread_end - thread_start
-                    print("Completed generating symbols for {}: elapsed time {} seconds".format(
-                        binary, elapsed.total_seconds()), flush=True)
-
-            q.task_done()
-
+    semaphore = asyncio.Semaphore(os.cpu_count())
+    list = []
```

Comment:
isn't `list` a built-in type name? should be something better here
```
>>> list()
[]
```

Reply:
Fixed